RELEASE_NOTES.md

```diff
@@ -8,10 +8,75 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
-## 12.19
+## 13.0.11 (2019-06-19)
 
+### Chef Server is now Chef Infra Server
 
-## 12.18
+Chef Server has a new name, but don’t worry, it’s the same Chef Server you’ve grown used to. You’ll notice new branding throughout the application and documentation but the command `chef-server-ctl` remains the same.
+
+### Chef EULA
+
+Chef Infra Server requires an EULA to be accepted by users before it can be installed. Users can accept the EULA in a variety of ways:
+
+- `chef-server-ctl reconfigure --chef-license accept`
+- `chef-server-ctl reconfigure --chef-license accept-no-persist`
+- `CHEF_LICENSE="accept" chef-server-ctl reconfigure`
+- `CHEF_LICENSE="accept-no-persist" chef-server-ctl reconfigure`
+
+Finally, if users run `chef-server-ctl reconfigure` without any of these options, they will receive an interactive prompt asking for license acceptance. If the license is accepted, a marker file will be written to the filesystem unless `accept-no-persist` is specified. Once this marker file is persisted, users no longer need to set any of these flags.
+
+See our [Frequently Asked Questions document](https://www.chef.io/bmc-faq/) for more information on the EULA and license acceptance.
+
+### Deprecation notice
+
+- Deprecated PowerPC and s390x platforms. (https://blog.chef.io/2018/11/01/end-of-life-announcement-for-chef-server-for-linux-on-ibm-z-and-linux-on-ibm-power-systems/)
+- Deprecated Keepalived/DRBD based HA. (https://blog.chef.io/2018/10/02/end-of-life-announcement-for-drbd-based-ha-support-in-chef-server/)
+- Deprecated Ubuntu 14.04 support. (Ubuntu 14 was EoL’d at the end of April 2019)
+
+### Updates and Improvements
+- Updated OpenResty to 1.13.6.2
+    - This fixes two CVEs CVE-2018-9230 and CVE-2017-7529.
+    - This version cannot be built on PowerPC and s390x because those platforms are not supported in mainline luajit.
+- Updated Ruby version to 2.5.5
+- Updated Chef Infra Client to 14.11.21
+- Updated runit to 5.1.1
+- Migrate unit tests from Travis to Buildkite. Reorganize them for improved speed, stability and portability.
+- Habitat:
+    - Added some Habitat packaging improvements with parameterized search_server.
```

Comment:
Is this the only bullet point for the Habitat updates?

Reply:
yes - looks like it is for that release.

Reply:
Just wanted to make sure. Thank you!